pkg/provider/branch/provider.go

```diff
@@ -270,27 +267,23 @@ func (b *branchENIProvider) UpdateResourceCapacity(instance ec2.EC2Instance) err
 
 // ReconcileNode reconciles a nodes by getting the list of pods from K8s and comparing the result
 // with the internal cache.
-func (b *branchENIProvider) ReconcileNode(nodeName string) (ctrl.Result, error) {
+func (b *branchENIProvider) ReconcileNode(nodeName string) bool {
 	trunkENI, isPresent := b.getTrunkFromCache(nodeName)
 	log := b.log.WithValues("node", nodeName)
 	if !isPresent {
-		log.Info("stopping the reconcile job")
-		return ctrl.Result{}, nil
+		log.Info("no trunk ENI is pointing to the given node", "NodeName", nodeName)
+		return true
```

Comment:
Why do we need to return `true` if no trunkENI found?

Reply:
Returning `true` means if we reconcile on this node again we use the shortest interval.